manifests/a/Araxis/Merge/2024.5981/Araxis.Merge.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS0.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: Araxis.Merge
+PackageVersion: "2024.5981"
```

```suggestion
PackageVersion: 70.30.5981
```
Please update file name and path to match this change.